README.md

```diff
@@ -19,3 +19,61 @@ file. If the file exists - it overrides main context config and config from  [`/
 
 **Note: keep this settings different from `/src/main/resources/config/application.yml`, 
 which is used for local run/debug, because the integration tests will remove all data form the CTP project**
+
+
+## How to use
+In this process, there are 3 parties involved:
+* The frontend -  the browser part of the shop. This is what the user interacts with.
+* The backend - the shop server.
+* The integration server - this is the place where this project commercetools-paypal-plus-integration is deployed and running. 
+
+1. In order to create a payment on PayPal Plus, backend must create first Payment object assigned to a cart. It must contain the following attributes:
+- amountPlanned
+- cancelUrl - this attribute is available from custom type. PayPal Plus redirects the user to this URL if he/she cancels the payment process.
+- successUrl - this attribute is available from custom type. PayPal Plus redirects the user to this URL if he/she successfully completes the payment process.
+- paymentMethodInfo needs to be set like this:
+```json
+   "paymentMethodInfo": {
+        "paymentInterface": "PAYPAL_PLUS",
+        "method": "paypal"
+      }
+```
+
+The backend calls the following URL of the integration server to obtain `approval_url`.
+```
+POST http://paypal-plus-integration-server.com/${projectName}/commercetools/create/payments/${ctpPaymentId}
+```
+This `approval_url` should be passed to frontend to generate a list of available payment methods. Example response:
+```
+{
+    "statusCode": 201,
+    "approvalUrl": "https://www.sandbox.paypal.com/cgi-bin/webscr?cmd=_express-checkout&token=${token}"
+}
+```
+
+2. After the user pays, he/she gets redirected to the `successUrl` defined in step 1. Additionally, PayPal Plus will add 3 request parameters to this URL.
```

Comment:
2) Execute payment after user successfully finished PayPal Plus checkout and was redirected back to the shop through `successUrl`. 

PayPal Plus will set 3 request parameters to `successUrl`:
- token
- paymentId - identifies this particular payment. **Required for execute payment.**
- PayerID - identifies the particular payer.  **Required for execute payment.**   

a) Backend extracts PayPal specific parameters: `paymentId`, `PayerID` and POSTs them to PayPal-Integration. Example:
...